frontend/Register/route.jsx

```diff
@@ -4,7 +4,7 @@ import Register from './index'
 
 const RegisterRoute = (props) => (
   <Route
-    path='/register2'
+    path='/registerNative'
```

Comment:
Application will have 2 register routes? `/register` and `registerNative` ? With different behavior ?

Reply:
that's right. the register route can't be changed at this point, because this is embedded pretty deep into the themes.